src/common/cluster-store.ts

```diff
@@ -211,6 +213,10 @@ export class ClusterStore extends BaseStore<ClusterStoreModel> {
     return this.activeCluster === id;
   }
 
+  isMetricHidden(resource: ResourceType) {
+    return clusterStore.getById(clusterStore.activeCluster).preferences.hiddenMetrics.includes(resource);
```

Comment:
We shouldn't be referencing an instance variable within the class, plus `active` might be `null` and `hiddenMetrics` might be `undefined`.
```suggestion
    return Boolean(this.active?.preferences.hiddenMetrics?.includes(resource));
```